docs/source/community/contributing.rst

```diff
@@ -2,100 +2,95 @@
+==========================================
+Ways to Contribute to Hyperledger Sawtooth
+==========================================
 
-Contributions by the community help grow and optimize the capabilities of
-Hyperledger Sawtooth, and are the most effective method of having a positive
-impact on the project.
+Contributions from the development community help improve the capabilities of
+Hyperledger Sawtooth. These contributions are the most effective way to
+make a positive impact on the project.
 
-**Different ways you can contribute**
+Ways you can contribute:
 
-* Bugs or Issues (issues or defects found when working with Sawtooth)
-* Core Features & Enhancements (expanded capabilities or optimization)
-* Arcade Features (games that demonstrate Sawtooth such as Tic-Tac-Toe
-  and Battleship)
-* New or Enhanced Documentation (improve existing documentation or create new)
-* Testing Events and Results (functional, performance or scalability)
+* Bugs or issues: Report problems or defects found when working with Sawtooth
+* Core features and enhancements: Provide expanded capabilities or optimizations
+* Arcade features: Add games that demonstrate Sawtooth; see the existing
+  games XO (tic-tac-toe) and Battleship
+* Documentation: Improve existing documentation or create new information
+* Tests for events and results: Add functional, performance, or scalability
+  tests
 
-**Unassigned JIRA Issues**
+Hyperledger Sawtooth issues can be found in :ref:`jira`.  Any unassigned items
+are probably still open. When in doubt, ask on RocketChat about
+a specific JIRA issue (see :doc:`join_the_discussion`).
 
-More specific items can be found in :ref:`jira`.  Any JIRA items which are
-unassigned are probably still open.  If in doubt, ask on :ref:`chat` about
-the particular JIRA issue.
-
-==============
-Commit Process
-==============
+==================
+The Commit Process
+==================
 
 Hyperledger Sawtooth is Apache 2.0 licensed and accepts contributions
 via `GitHub <https://github.com/hyperledger/sawtooth-core>`_
-pull requests. When contributing code please do the following:
+pull requests. When contributing code, please follow these guidelines:
 
-* Fork the repository and make your changes in a feature branch.
-* Please include unit and integration tests for any new features and updates
-  to existing tests.
-* Please ensure the unit and integration tests run successfully. Both are run
-  with `./bin/run_tests`.
-* Please ensure that lint passes by running './bin/run_lint -s master'.
-  On success, the command produces no output.
+* Fork the repository and make your changes in a feature branch
+* Include unit and integration tests for any new features and updates
+  to existing tests
+* Ensure that the unit and integration tests run successfully. Run both
+  of these tests with ``./bin/run_tests``
+* Check that the lint tests pass by running ``./bin/run_lint -s master``.
+  Note that this command does not produce output on success
```

Comment:
This is no longer true. Some output is printed on success.

Reply:
Fixed by deleting this line.